web/impact/impact/urls.py

```diff
@@ -42,37 +42,35 @@
 simpleuser_router = routers.DefaultRouter()
 simpleuser_router.register('User', GeneralViewSet, base_name='User')
 
-for model in apps.get_models('accelerator'):
-    if (model._meta.app_label == 'accelerator' and not
-            model._meta.auto_created and
-            model.__name__ not in MODELS_TO_EXCLUDE_FROM_URL_BINDING):
-        schema_router.register(
-            model, url=model_name_to_snake(model.__name__))
+if len(schema_router.registry) == 0:
```

Comment:
this is needed because it seems that in the context of running impact in accelerate, the urls file gets loaded by the interpreter twice and so a double-registration of these views happen.